codemeta.json

```diff
@@ -0,0 +1,83 @@
+{
+    "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
+    "@type": "SoftwareSourceCode",
+    "license": "https://spdx.org/licenses/GPL-3.0",
+    "codeRepository": "https://github.com/javierrico/gLike",
+    "dateCreated": "2018-11-13",
+    "datePublished": "2020-07-30",
+    "dateModified": "2021-03-12",
```

Comment:
What do these dates refer to?

Reply:
Sorry, used your full name

Reply:
They are the creation and first release date, see "The software itself" section of the [codemeta generator](https://codemeta.github.io/codemeta-generator/). 
I obtained the creation date, `dateCreated`, from the GitHub repos API (just goggle https://api.github.com/repos/javierrico/glike).
And the date of the first release `datePublished`, simply going through your releases in GitHub.